VERSIONS.md

```diff
@@ -14,4 +14,4 @@
 | 2.3.3   | 2.5.0       | 2.3.1           | 0.1.2                |
 | 2.3.2   | 2.5.0       | 2.3.0           | 0.1.2                |
 | 2.3.1   | 2.5.0       | 2.3.0           | 0.1.1                |
-| 2.3.0   | 2.4.0       | 2.3.0           | N/A                  |
\ No newline at end of file
+| 2.3.0   | 2.4.0       | 2.3.0           | N/A                  |
```

Comment:
Why is this a change lol

Reply:
some weird leftover from when I had actually changed the version number. I'll roll it back. My editor is set to always add an empty line at EoF